ci-operator/step-registry/ipi/install/powervs/install/ipi-install-powervs-install-commands.sh

```
@@ -935,8 +937,11 @@ export TF_LOG=debug
 
 echo "8<--------8<--------8<--------8<-------- BEGIN: create cluster 8<--------8<--------8<--------8<--------"
 echo "DATE=$(date --utc '+%Y-%m-%dT%H:%M:%S%:z')"
-openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-buffered -v 'password\|X-Auth-Token\|UserData:'
-ret=${PIPESTATUS[0]}
+# Run openshift-install in background with process substitution to allow trap to execute on SIGTERM
+openshift-install --dir="${dir}" create cluster 2>&1 > >(grep --line-buffered -v 'password\|X-Auth-Token\|UserData:') &
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)